.github/workflows/build_changed_packages.yml

```
@@ -0,0 +1,97 @@
+name: Build changed packages
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: jitterbit/get-changed-files@v1
+        id: files
+        with:
+          format: space-delimited
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Find changed packages
+        run: |
+          function list_include_item {
+            local list="$1"
+            local item="$2"
+            if [[ $list =~ (^|[[:space:]])"$item"($|[[:space:]]) ]] ; then
+              result=0
+            else
+              result=1
+            fi
+            return $result
+          }
+          CHANGED_PACKAGES=()
+          for changed_file in ${{ steps.files.outputs.all }}; do
+            if [[ "$changed_file" == "packages"*".cc" ]] || [[ "$changed_file" == "packages"*".h" ]]; then
+                package=$(echo ${changed_file} | sed 's|\/tizen.*||;s|.*packages\/||')
+                if ! list_include_item $CHANGED_PACKAGES $package; then
+                    CHANGED_PACKAGES+=($package)
+                fi
+            fi
+          done
+          if [[ ${#CHANGED_PACKAGES[@]} -eq 0 ]]; then
+              echo "ENABLE=false" >> "$GITHUB_ENV"
+          else
+              echo "ENABLE=true" >> "$GITHUB_ENV"
+              echo "CHANGED_PACKAGES=${CHANGED_PACKAGES[@]}" >> "$GITHUB_ENV"
+          fi
+      - name: Install prerequisite packages
+        if: contains(env.ENABLE, true)
```

How about
```suggestion
        if: env.ENABLE == 'true'
```
or just
```suggestion
        if: env.CHANGED_PACKAGES != ''
```

---

Thanks~ I choose 2nd